library/core/src/cell.rs

```diff
@@ -333,7 +333,8 @@ impl<T: Copy> Clone for Cell<T> {
 }
 
 #[stable(feature = "rust1", since = "1.0.0")]
-impl<T: Default> Default for Cell<T> {
+#[rustc_const_unstable(feature = "const_default", issue = "67792")]
+impl<T: ~const Default> const Default for Cell<T> {
```

Comment:
```suggestion
impl<T: [const] Default> const Default for Cell<T> {
```

Reply:
The parser changes landed. But rustfmt is bumped when the bootstrap compiler is bumped. At that time everything will automatically be formatted from `~const` to `[const]`